iAnnotateSV/AnnotateEachBreakpoint.py

```diff
@@ -166,4 +168,6 @@ def AnnotateEachBreakpoint(chromosome,position,strand,df,autoSelect):
                 desc = 'IGR: ' + hp.bp2str(distBefore[afterIdx],2) + ' after ' + geneName + '(' + strandDirection + ')' 
                 #print a
                 break
-    return(geneName,transcript,desc,zone,strandDirection,intronnum,intronframe)   
```

Comment:
Thanks for addressing this. Do we want to raise an error here or return some sort of dummy value.

Just wondering because for purposes of pipeline processing one sample can hold up many others if this happens.

Reply:
@ionox0  I assume you are referring to the error raised here: https://github.com/rhshah/iAnnotateSV/pull/31/files/f6b75a47720939503159df5d9330d484df8bba2e#diff-b741ce66ea22017a44e20b835e8f3262R20

This error, if raised, is caught and handled in the main iAnnotateSV.py module, and precisely to address the concern you've raised - not to hold up pipeline processing for a project because of encountering this error for one sample. https://github.com/rhshah/iAnnotateSV/blob/ae8b4f1418dc5103c87aea2ea90f7ccf10330211/iAnnotateSV/iAnnotateSV.py#L286-L292